atom/browser/native_window_views.cc

```diff
@@ -1090,9 +1090,31 @@ void NativeWindowViews::OnWidgetBoundsChanged(
   if (widget != window_.get())
     return;
 
-  if (widget_size_ != bounds.size()) {
+  // Note: We intentionally use `GetBounds()` instead of `bounds` to properly
+  // handle minimized windows on Windows.
+  const auto new_bounds = GetBounds();
+  if (widget_size_ != new_bounds.size()) {
+    if (browser_view_) {
+      const auto flags = static_cast<NativeBrowserViewViews*>(browser_view_)
```

Comment:
This change looks good to me, just wondering if maybe the code inside this `if (browser_view_)` block should be moved into a helper method in `NativeBrowserViewViews` instead of living in `native_window_views.cc`, something like `NativeBrowserViewViews:: OnWidgetBoundsChanged` that just passes in the `new_bounds` as a param.